src/silicone/utils.py

```diff
@@ -464,6 +470,16 @@ def _adjust_time_style_to_match(in_df, target_df):
     return in_df
 
 
+def _remove_equivs(string_to_fix):
+    """
+    Removes the substring "-equiv" from strings. For use in unit conversion
+    :param string_to_fix: str
```

Comment:
```suggestion
Parameters
-----------
string_to_fix : str

Returns
--------
str
```

Just for consistency

Reply:
Preferable